llama_stack/providers/inline/agents/meta_reference/responses/utils.py

```diff
@@ -146,6 +157,8 @@ async def convert_response_input_to_chat_messages(
                         f"Llama Stack OpenAI Responses does not yet support message role '{input_item.role}' in this context"
                     )
                 messages.append(message_type(content=content))
+        for result in tool_call_results.values():
```

Comment:
should we do this? if you found an output for which there is no func call, that just seems odd

Reply:
I've added a commit that raises an error if there are any function_call_outputs without a matching function_call. I can squash this into the first commit if you agree with that approach.